doc/parameters.md

```diff
@@ -28,6 +28,25 @@ Specifies the period (in seconds) that the configuration will be stored in the c
 
 This parameter is also used to store OpenID discovery configuration in the local cache, as the same behavior as described above.
 
+### `APICAST_CONFIGURATION_CACHE_STALE`
+
+**Values:** _a positive number_
+**Default:** 0
+
+When configuration cannot be gather correctly due to network errors, or any
```

Comment:
Suggestion:
_You can use this variable for the specified number of times to gather configuration details, so requests will not fail due to not found services. This variables is useful when the configuration cannot be correctly obtained due to network errors, or any other issues such as an old configuration stored in the cache._